### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by stephenkati**
- [ ] Created JSON files to store data for:
- Rentals
- Books
- People

- [ ] Read data from files using ruby
- [ ] Store data in files using ruby

### CODE REVIEW

File: preserve_data.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  else
+    []
+  end
+end
+
+def file_write(file, content)
+  hashed_content = content.map(&:to_hash)
+  json = JSON.generate(hashed_content)
+  File.write(file, json)
+end
```
Comment (julie-ify) [inline preserve_data.rb]: - [ ] There is an issue with your save data process. When I add data it overwrites the existing data in the storage files. The json files only work with data that was created when the app runs. When the app is re-run and new data is created, the previous data is cleared. Kindly fix that. The new data should be appended to the existing data.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @stephenkati and @ehmaddd,

Good job so far!

## Highlights!
- Good code organization and storage file creation ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: preserve_data.rb (hunk 1 above)
Comment (stephenkati) [reply preserve_data.rb]: Thanks @julie-ify, I will make that change

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @stephenkati and @ehmaddd, 

To highlight 👍🏽 
✔️ App runs smoothly
✔️ Data is saved

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**